Automattic/jetpack
# Search: Migrate helper classes to standalone package

#### Changes proposed in this Pull Request:
* Moves the Helper and Options classes from the Jetpack plugin in a backward-compatible manner.
* Modifies syntax of existing invocations of the helper classes in the Jetpack plugin.

#### Jetpack product discussion
Part of the standalone Search package/plugin project.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Apply this change to your Jetpack site.
* Ensure that the following features continue working as expected:
  * [x] Classic Search
  * [x] Instant Search
  * [x] Jetpack Search section of the Customizer 
  * [x] Customberg
